lib/maropost/api.rb

```diff
@@ -1,93 +1,86 @@
 module Maropost
   class Api
-    def self.find(email)
-      response = request(:get,
-                         maropost_url('contacts/email.json', "contact[email]=#{CGI::escape(email)}"))
-      Maropost::Contact.new(JSON.parse response.body)
-    rescue RestClient::ResourceNotFound
-      nil
-    end
+    class << self
```

Comment:
do you prefer this to keep class methods contained in a block?

Reply:
I personally prefer it because it allows private class-level methods:

```ruby
class Foo

  private

  def self.bar
    'called!'
  end
end

Foo.bar # => "called!"
```

vs.

```ruby
class Foo
  class << self
    private
    
    def bar
      'called!'
    end
  end
end

Foo.bar # => NoMethodError: private method `bar' called for Foo:Class
```